src/tiled/stampbrush.cpp

```diff
@@ -56,6 +56,7 @@ StampBrush::StampBrush(QObject *parent)
     , mIsRandom(false)
     , mIsWangFill(false)
     , mWangSet(nullptr)
+    , mRandomCacheValid(false)
```

Comment:
I believe the value should initially be `true`. Since `mStamp` will be an empty stamp, the initially empty random list is valid.